api/apps/geoprocessing/src/migrations/geoprocessing/1692254627921-AddsCostSurfacePuDataGeoDb.ts

```diff
@@ -9,9 +9,14 @@ export class AddsCostSurfacePuDataGeoDb1692254627921
     await queryRunner.query(`
       CREATE TABLE cost_surface_pu_data (
         id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+        -- consider marking this as not null, as it is for projects, and maybe required for the GC
```

Comment:
We have to make clear then that we will fill the cost_surface_id with something random (the_geom_id) as there are use cases where we don't know the api.cost_surfaces.id by the time we insert these values.

It is what we are doing now.

Reply:
just remember to remove the comment (which I think you meant to be `consider marking this as null`, as in `projects_pu.project_id`), either way - it's already confusing enough as is because of reasons 😓